pandas/core/groupby/groupby.py

```diff
@@ -438,7 +438,7 @@ def get_converter(s):
                     return [self.indices[name] for name in names]
                 except KeyError:
                     # turns out it wasn't a tuple
-                    msg = ("must supply a a same-length tuple to get_group"
+                    msg = ("must supply a same-length tuple to get_group"
```

Comment:
be aware that I have changed an error message in the codebase

Reply:
And this is why we should be checking error messages if possible, to catch things like this 🙂